eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/properties/CdcPipelineProperties.java

```diff
@@ -53,4 +62,44 @@ public String getSourceTableName() {
   public void setSourceTableName(String sourceTableName) {
     this.sourceTableName = sourceTableName;
   }
+
+  public Boolean getPurgeMessagesEnabled() {
```

Comment:
Why are these here?
Cleaner is nothing to do with pipeline

Reply:
yes, missed to remove here